src/test/java/alinaTinca/tests/Tema5/Demoqa/Elements/Links.java

```diff
@@ -0,0 +1,35 @@
+package alinaTinca.tests.Tema5.Demoqa.Elements;
+
+import AlinaTinca.BrowserManager;
+import AlinaTinca.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Links {
+
+    public static void main(String[] args) {
+        goToTheLinks();
+        //check if the method called goToTheLinks is done successfully
+    }
+
+    public static void goToTheLinks(){
+        //create a method called goToTheLinks that find a link and it clicks on it
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        //create and configure the Chrome browser page settings
+        driver.get("https://demoqa.com/links");
+        //go to this URL
+        WebElement linksSection = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[6]"));
```

Comment:
Try to scroll first to this section as it's covered by the add. The exception is -> 
Exception in thread "main" org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <li class="btn btn-light active" id="item-5">...</li> is not clickable at point (133, 574). Other element would receive the click: <iframe id="google_ads_iframe_/21849154601,22343295815/Ad.Plus-Anchor_0" name="google_ads_iframe_/21849154601,22343295815/Ad.Plus-Anchor_0" title="3rd party ad content" width="970" height="100" scrolling="no" marginwidth="0" marginheight="0" frameborder="0" role="region" aria-label="Advertisement" tabindex="0" sandbox="allow-forms allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-top-navigation-by-user-activation" style="border: 0px; vertical-align: bottom;" data-load-complete="true" data-google-container-id="1"></iframe>

Reply:
I do not understand what you mean with this message

Reply:
That the exception from console due to the fact that there is an add that appear and covers the element sometimes. So it's safer to scroll first to the element and after that interacting with it.